reconcile/terraform_aws_route53.py

```diff
@@ -75,6 +76,11 @@ def build_desired_state(
             # Process '_target_cluster'
             target_cluster = record.pop("_target_cluster", None)
             if target_cluster:
+                if not integration_is_enabled(
+                    integration=QONTRACT_INTEGRATION.replace("_", "-"),
+                    disable_obj=target_cluster,
+                ):
+                    continue
```

Comment:
Please add log.info message, otherwise, it would be hard to see why the integration doesn't do anything for a specific cluster

Reply:
Fixed